cmake/modules/FindUFE.cmake

```diff
@@ -82,6 +83,13 @@ find_library(UFE_LIBRARY
     NO_DEFAULT_PATH
 )
 
+# Gather all preview features that might be there or not into a single list:
+list(APPEND UFE_PREVIEW_FEATURES v4)
```

Comment:
We can't just say that it includes v4, because when compiling with older versions of Maya we aren't using Ufe v4.

Reply:
Isn't this rather saying that UFE_PREVIEW_FEATURES is a list that pertains to UFE v4?  Not sure what we're getting by placing this "v4" string into the list, because it will always be the case that preview features will pertain to the up-coming UFE version.

Reply:
In fact, I would push one step further and say that maybe even "preview" is too specific. If we imagine this being only a list of Ufe features, then it enable them as soon as Ufe supports them in Maya, which becomes very interesteing from a feature backporting scenario. I do remember the MAYA_LIGHT_API_VERSION kicking in when I backported the light API fixes for 2020.3 without having to update MayaUSD.

Might be potential discussion subject for the UFE interest meeting.

Reply:
Better initializer, since list can not be empty, would be "ufe".